utils/TestUtils.ts

```diff
@@ -123,9 +98,10 @@ export class ValkeyCluster {
     }
 
     public static async initFromExistingCluster(
-        addresses: [string, number][]
+        addresses: [string, number][],
+        getVersionCallback: (addresses: [string, number][]) => Promise<string>
```

Comment:
You need to add `cluster_mode: boolean` arg to `initFromExistingCluster` and properly set it

Reply:
updated.